src/rest_api/classes/generic_functions.clj

```diff
@@ -22,3 +24,44 @@
 
     (contains? h :qualifier/partial)
     "Partial"))
+
+(defn- is-cgc?  [strain]
+  (some #(=  (->>  (:strain.location/laboratory %)
```

Comment:
There are several places where you used double space. Is this intentional?

Reply:
This is not intentional but I know what causes it. If I cur and paste code into vim it will try to format it and for some reason changes it to double space. If I turn off the formatting and then paste it doesn't to id. I must have pasted this in from somewhere else without turning off vims formatting.